bundle/kubernetes/0.1.0/wavefront-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,295 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "wavefront.com/v1alpha1",
+          "kind": "WavefrontProxy",
+          "metadata": {
+            "name": "wavefront-proxy",
+            "namespace": "wavefront"
+          },
+          "spec": {
+            "token": "\u003cTOKEN\u003e",
+            "url": "\u003chttps://YOUR_CLUSTER.wavefront.com/api/\u003e",
+            "image": "wavefronthq/proxy:5.7",
+            "enableAutoUpgrade": true
```

Comment:
By default enableAutoUpgrade should be false and users can enable it as needed. 
So can just remove this parameter from here. we don't support upgrade for openshift yet anyways.